platform/mbed_sleep.h

```diff
@@ -64,6 +64,34 @@ extern "C" {
  * }
  * @endcode
  */
+#ifdef MBED_SLEEP_STATS_ENABLED
+
+#define sleep_manager_lock_deep_sleep()           \
```

Comment:
How deep is sleep for peripherals at present? Do we disable clock for UART in any target device?

Reply:
It's sort of the other way around, if you use serial then the deep sleep is disabled. So this info is theoretical and @scartmell-arm will create a docs how to interpret that data.